build-scripts/search-index.ts

```diff
@@ -17,7 +17,9 @@ const createPostIndex = () => {
 			{
 				name: "authorName",
 				getFn: (post) => {
-					return post.authorsMeta.map((author) => author.name).join(", ");
+					return post.authors
+						.map((id) => api.getUnicornById(id, post.locale))
+						.join(", ");
```

Comment:
```suggestion
					return post.authors
						.map((id) => api.getUnicornById(id, post.locale)!.name)
						.join(", ");
```
This looks like it's missing a `.name` prop

Reply:
Looking at the built index, I think `authorHandles` needs inspecting as well - seems to generate a lot of empty/duplicate items i.e. `"crutchcorn, crutchcorn, crutchcorn, , , , , "`.

Reply:
Fixed